go.mod

```diff
@@ -39,7 +39,7 @@ require (
 	github.com/stretchr/testify v1.8.2
 	github.com/tangzero/inflector v1.0.0
 	github.com/withfig/autocomplete-tools/packages/cobra v1.2.0
-	go.mongodb.org/atlas v0.26.2-0.20230516134051-330b8d7102e7
+	go.mongodb.org/atlas v0.25.1-0.20230518112041-ac1214205bd5
```

Comment:
this seems like a regression

Reply:
Ah I'd merged master changes back to sdkv2dev instead of sdkv2, so when I generated again, it wiped the merge

Raised this PR to get latest. Will regenerate again once merged https://github.com/mongodb/go-client-mongodb-atlas/pull/484

Reply:
FYI the branch you are using here seems to still be behind master which is what I'm seeing 0.26.2 != 0.25.1, the branch must include the commit that was tagged 0.26.2

Reply:
I think this is because of squash on merge
I guess the individual commit should've been backported separately?
I'll push another change to get the version to update

Reply:
do a simple fir merge to the branch from master

Reply:
Merged latest master to sdkv2dev and seems ok